reference/conanfile/methods.rst

```diff
@@ -1,2 +1,130 @@
+.. spelling::
+
+  ing
+  ver
+
+.. _conan_conanfile_methods:
+
 Methods
-=======
\ No newline at end of file
+=======
+
+
+requirements()
```

Comment:
I added no autodocs, just leaving this here so we don't lose it, it may go to the dosctring in the future.